trunk/SUAVE/Methods/Propulsion/Rotor_Wake/Fidelity_One/compute_wake_induced_velocity.py

```diff
@@ -29,68 +30,68 @@ def compute_wake_induced_velocity(WD,VD,cpts,sigma=0.11):
+    YC    = np.tile(VD.YC.astype(dtype)[None,None,:],(cpts,num_vortex_pts,1))
+    ZC    = np.tile(VD.ZC.astype(dtype)[None,None,:],(cpts,num_vortex_pts,1))
     
     # -------------------------------------------------------------------------------------------
     # Compute velocity induced by horseshoe vortex segments on every control point by every panel
     # -------------------------------------------------------------------------------------------     
     # Create empty data structure
     V_ind = np.zeros((cpts,VD.n_cp,3))
      
-    #compute vortex strengths for every control point on wing 
-    # this loop finds the strength of one ring only on entire control points on wing 
     # compute influence of bound vortices 
-    _ , res_C_AB = vortex(XC, YC, ZC, WXA1, WYA1, WZA1, WXB1, WYB1, WZB1,sigma,GAMMA,bv=True,VD=VD) 
-    C_AB         = np.transpose(res_C_AB,axes=[1,2,3,0]) 
+    _ , res_C_AB = vortex(XC, YC, ZC, WXA1, WYA1, WZA1, WXB1, WYB1, WZB1,sigma,GAMMA,bv=True,WD=WD) 
+    C_AB         = res_C_AB.transpose(1,3,0,2) 
     
-    # compute influence of 3/4 right legs 
-    _ , res_C_BC = vortex(XC, YC, ZC, WXB1, WYB1, WZB1, WXB2, WYB2, WZB2,sigma,GAMMA) 
-    C_BC         = np.transpose(res_C_BC,axes=[1,2,3,0]) 
+    # compute influence of right vortex segment
+    _ , res_C_BC = vortex(XC, YC, ZC, WXB1, WYB1, WZB1, WXB2, WYB2, WZB2,sigma,GAMMA)
+    C_BC         = res_C_BC.transpose(1,3,0,2) 
     
-    # compute influence of bound vortices  
+    # compute influence of bottom vortex segment
     _ , res_C_CD = vortex(XC, YC, ZC, WXB2, WYB2, WZB2, WXA2, WYA2, WZA2,sigma,GAMMA) 
-    C_CD         = np.transpose(res_C_CD,axes=[1,2,3,0])
+    C_CD         = res_C_CD.transpose(1,3,0,2) 
     
-    # compute influence of 3/4 left legs  
+    # compute influence of left vortex segment 
     _ , res_C_DA = vortex(XC, YC, ZC, WXA2, WYA2, WZA2, WXA1, WYA1, WZA1,sigma,GAMMA) 
-    C_DA         = np.transpose(res_C_DA,axes=[1,2,3,0]) 
+    C_DA         = res_C_DA.transpose(1,3,0,2) 
     
     # Add all the influences together
-    V_ind =  np.sum(C_AB +  C_BC  + C_CD + C_DA, axis = 1)
+    V_ind =  row_reduction_summation(C_AB) + row_reduction_summation(C_BC)  + row_reduction_summation(C_CD) + row_reduction_summation(C_DA)   
     
+
     return V_ind
   
   
 # -------------------------------------------------------------------------------
 # vortex strength computation
 # -------------------------------------------------------------------------------
 ## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
-def vortex(X,Y,Z,X1,Y1,Z1,X2,Y2,Z2,sigma, GAMMA = 1, bv=False,VD=None,use_regularization_kernal=True):
+def vortex(X,Y,Z,X1,Y1,Z1,X2,Y2,Z2,sigma, GAMMA = 1, bv=False,WD=None,use_regularization_kernal=True):
     """ This computes the velocity induced on a control point by a segment
-    of a horseshoe vortex from point 1 to point 2 
+    of a horseshoe vortex that points from point 1 to point 2 
```

Comment:
the vortex doesn't necessarily point from 1 to point 2

Reply:
it does for a positive vortex strength. Added this distinction to the comment